tests/views/timeline.py

```diff
@@ -67,14 +69,35 @@ def _get_suggestion_description(submission):
     display_name = (
         submission["suggestion__user__full_name"].strip()
         if submission["suggestion__user__full_name"].strip()
-        else submission["suggestion__user__username"])
-    author = format_html(u'<a href="{}">{}</a>', user_url, display_name)
+        else submission["suggestion__user__username"].strip())
+    params = {
+        'author': format_html(u'<a href="{}">{}</a>',
+                              user_url,
+                              display_name)
+    }
+    try:
+        comment = get_model().objects \
+                             .for_model(Suggestion) \
+                             .get(object_pk=submission["suggestion_id"])
+        params.update({
+            'comment': format_html(u'<span class="comment">{}</span>',
+                                   comment.comment),
+        })
```

Comment:
This second statement can live in the `else` clause of the `try` block. Helps us spotting which is the exact code that we are enclosing to catch exceptions.